src/System.IO.Pipes/ref/System.IO.Pipes.cs

```diff
@@ -116,22 +116,22 @@ public abstract partial class PipeStream : System.IO.Stream
         protected internal virtual void CheckPipePropertyOperations() { }
         protected internal void CheckReadOperations() { }
         protected internal void CheckWriteOperations() { }
-        protected bool IsHandleExposed { get { throw null; } }
         protected override void Dispose(bool disposing) { }
         public override int EndRead(System.IAsyncResult asyncResult) { throw null; }
-        public override void EndWrite(System.IAsyncResult asyncResult) { throw null; }
+        public override void EndWrite(System.IAsyncResult asyncResult) { }
         public override void Flush() { }
+        public override System.Threading.Tasks.Task FlushAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
Should we add this API to the ref?

Reply:
Since the type is sealed it doesn't matter a whole lot, seems fine to let it be added.

Reply:
Yeah, when I was going through and fixing up virtual overrides previously, I didn't bother with ones on sealed types.  I don't have a strong preference.